deploy/samconfig.toml

```
@@ -0,0 +1,11 @@
+version = 0.1
+[dev]
+[dev.deploy]
+[dev.deploy.parameters]
+stack_name = "backend-viveak"
```

```suggestion
stack_name = "backend-main"
```
This makes it pretty clear where we're sourcing the version from.